lib/mail-archive-helper.ts

```diff
@@ -117,8 +117,9 @@ export class MailArchiveGitHelper {
             if (!sousChef.messageID) {
                 throw new Error(`Could not parse Message-ID of ${mbox}`);
             }
+            console.log(`Handling "${sousChef.subject}"`);
             const whatsCookingBaseURL = "https://lore.kernel.org/git/";
-            for (const branchName of Object.keys(sousChef.branches)) {
+            for (const branchName of sousChef.branches.keys()) {
```

Comment:
Sorry I missed this (I had updated in my local test branch).  May need to look at creating tests for `MailArchiveGitHelper`.

Reply:
That would be nice. I am not quite comfortable yet adding these tests...